net-firewall/shorewall/files/4.6/shorewall-lite.systemd-r1

```diff
@@ -0,0 +1,20 @@
+#
+#	The Shoreline Firewall Lite (Shorewall-Lite) Packet Filtering Firewall - V4.6
+#
+[Unit]
+Description=Shorewall IPv4 firewall lite
+Documentation=man:shorewall-lite(8) http://www.shorewall.net/Documentation_Index.html
+Wants=network-online.target
+After=network-online.target
+Conflicts=iptables.service firewalld.service
```

Comment:
In Gentoo, `iptables.service` is just a wrapper. Look at how we do firewall units. If shorewall's not using a daemon, you likely want to copy that concept.